wheel/build_wheel_manylinux.sh

```diff
@@ -1,11 +1,19 @@
 #!/usr/bin/env bash
 
+set -ex
+
 source /multibuild/manylinux_utils.sh
 
+TVM_PYTHON_DIR="/workspace/tvm/python"
+PYTHON_VERSIONS_CPU=("3.7" "3.8" "3.9" "3.10")
+PYTHON_VERSIONS_GPU=("3.7" "3.8" "3.9" "3.10")
```

Comment:
This doesn't match the Python versions installed in the Docker image proposed here.

Reply:
I updated the install scripts to use the miniconda Pythons (and detect them here instead of manually listing them out) which I think makes this all a bit cleaner and easier to update in the future when 3.11 rolls around